src/Orchestration/Adapter/DockerCLI.php

```diff
@@ -120,19 +120,19 @@ public function run(string $image, string $name, array $command, string $entrypo
         $stdout = '';
         $stderr = '';
 
-        foreach ($command as &$value) {
+        foreach ($command as $key => $value) {
             if (str_contains($value, " ")) {
-                $value = "'".$value."'";
+                $command[$key] = "'".$value."'";
             }
         }
 
         $labelString = ' ';
 
-        foreach ($vars as $key => &$value) {
+        foreach ($vars as $key => $value) {
             $key = $this->filterEnvKey($key);
 
-            $value = \escapeshellarg((empty($value)) ? '' : $value);
-            $value = "--env {$key}={$value}";
+            $vars[$key] = \escapeshellarg((empty($value)) ? '' : $value);
+            $vars[$key] = "--env {$key}={$value}";
```

Comment:
This won't work anymore - since they are not referenced anymore :-)

Reply:
```suggestion
            $value = \escapeshellarg((empty($value)) ? '' : $value);
            $vars[$key] = "--env {$key}={$value}";
```

like this it would escape again 👍🏻